user_tools/custom_speedup_factors/validate_qualification_estimates.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Just wanted to check if we need to add this(SPDX-FileCopyrightText) copyright for new files going forward? Earlier files had different format. So curious to know if this is required.

Reply:
Probably my fault.  I think I copied it from the spark-rapids-benchmarks repo which includes TPC-DS info for license.  I can update license for the Python scripts to use the standard Apache License we have in our tools source code.

Reply:
Addressed in latest commit